company/values.md

```diff
@@ -1,15 +1,34 @@
 # Sourcegraph values
 
-Our values are:
+Our values reflect who we are and how we work.
 
-## People
+## People first
```

Comment:
This paragraph mentions both internal and external context of people. 
Does it make sense to move 
> Together we are advancing technology for the good of people all around the world

to "Customer driven progress"?
And orient this value more "internally"? 
If I understand correctly, values should be designed to be a guiding star in decision making process. If so, I feel like, the description here doesn't provide clear guidance how to prioritize trade offs. Does it make sense?

Reply:
Also maybe mentions something along the lines:
> we value and leverage our strengths, celebrate differences and be kind to our weaknesses.

Maybe a bit to poetic, but I would love to convey a meaning "it is ok to be a human". I think it nicely connects to  our mission aspect of "sourcegraph is for everybody", thus we need/should to acknowledge that everybody is different